### PR TITLE
Added missing features from NG to feature/deployment

### PR DESCRIPTION
**Pull request by mkurek**
- Filtering by service uid in API
- Networking info for cloud host in inline

### CODE REVIEW

File: src/ralph/virtual/admin.py [hunk 1]
```diff
@@ -109,10 +110,20 @@ def get_queryset(self, *args, **kwargs):
         ).prefetch_related('tags')
 
 
+class CloudHostNetworkForm(SimpleNetworkForm):
```
Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

File: src/ralph/virtual/admin.py [hunk 2]
```diff
@@ -109,10 +110,20 @@ def get_queryset(self, *args, **kwargs):
...
+    class Meta(SimpleNetworkForm.Meta):
```
Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Too few public methods \(0/2\) \[too\-few\-public\-methods\]

File: src/ralph/virtual/admin.py (hunk 1 above)
Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

File: src/ralph/virtual/admin.py (hunk 2 above)
Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Too few public methods \(0/2\) \[too\-few\-public\-methods\]

File: src/ralph/virtual/admin.py (hunk 1 above)
Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

File: src/ralph/virtual/admin.py (hunk 2 above)
Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Missing class docstring \[missing\-docstring\]

Comment (sputnikci) [inline src/ralph/virtual/admin.py]: \[Pylint\] INFO: Too few public methods \(0/2\) \[too\-few\-public\-methods\]

**Comment by ar4s** [comment]
:ok:

**Pull request merged**